content/post/125-go-ipfs-0-8-0.md

```diff
@@ -0,0 +1,109 @@
+---
+date: 2021-02-19
+url: /2021-02-19-go-ipfs-0-6-0/
+title: go-ipfs 0.8.0, and Remote Pinning, is here!
+author: Adin Schmahmann
+tags: ipfs, go-ipfs, release notes
+header_image: 125-go-ipfs-0-8-0.png
+snippet: See what's new in go-ipfs 0.8.0 (spoiler - pins are easier)!
+---
+
+## What's new in go-ipfs 0.8.0? 
+
```

Comment:
Should we add a note about pinning services that have implemented this pinning services api and IPFS desktop which will also get new capabilities? Moar images pls!!

Reply:
Emily is aware of this and will supply changed content.

Reply:
@momack2 - https://github.com/ipfs/blog/pull/519/commits/f136c53e1b4413aa4aa5d34d0bd01ce30ab67d47 adds a note on Desktop/Web UI plans forthcoming and https://github.com/ipfs/blog/pull/519/commits/8c6b660c187c4c81d65232372d16769d3d6a6d5d mentions Pinata support.